adguard/list_filter_resource_test.go

```diff
@@ -41,11 +41,13 @@ resource "adguard_list_filter" "test_blacklist" {
 resource "adguard_list_filter" "test_blacklist" {
   name = "Test Blacklist Filter Updated"
   url  = "https://adguardteam.github.io/HostlistsRegistry/assets/filter_5.txt"
+  enabled   = false
```

Comment:
I don't fully understand why you changed this? Can you please elaborate?

Reply:
<img width="1082" alt="image" src="https://github.com/gmichels/terraform-provider-adguard/assets/8067559/291a6211-e7b1-4c52-aaef-34f5a96ef84a">
Without `enable=false`, if locally run `make testacc` I got a failed test because `adguard` was not ready for requests (`set_url`) because applying all rules, but this simple change helped me and did not break the test assertion

Reply:
I can't reproduce your behavior, it works just fine for me.

```
➜  terraform-provider-adguard git:(main) make testacc
docker compose -f ./docker/docker-compose.yaml up -d
WARN[0000] /Users/gmichels/git/terraform-provider-adguard/docker/docker-compose.yaml: `version` is obsolete 
[+] Running 2/2
 ✔ Network docker_default  Created                                                                                                                                               0.0s 
 ✔ Container adguardhome   Started                                                                                                                                               0.2s 
TF_ACC=1 go test $(go list ./adguard | grep -v 'vendor') -v  -timeout 10m
=== RUN   TestAccClientDataSource
--- PASS: TestAccClientDataSource (1.08s)
=== RUN   TestAccClientResource
--- PASS: TestAccClientResource (2.26s)
=== RUN   TestAccConfigDataSource
--- PASS: TestAccConfigDataSource (4.53s)
=== RUN   TestAccConfigResource
--- PASS: TestAccConfigResource (7.90s)
=== RUN   TestAccListFilterDataSource
--- PASS: TestAccListFilterDataSource (1.46s)
=== RUN   TestAccListFilterResource
--- PASS: TestAccListFilterResource (7.87s)
=== RUN   TestAccRewriteDataSource
--- PASS: TestAccRewriteDataSource (0.78s)
=== RUN   TestAccRewriteResource
--- PASS: TestAccRewriteResource (1.74s)
=== RUN   TestAccUserRulesDataSource
--- PASS: TestAccUserRulesDataSource (0.78s)
=== RUN   TestAccUserRulesResource
--- PASS: TestAccUserRulesResource (1.45s)
PASS
ok      github.com/gmichels/terraform-provider-adguard/adguard  30.137s
docker compose -f ./docker/docker-compose.yaml down
WARN[0000] /Users/gmichels/git/terraform-provider-adguard/docker/docker-compose.yaml: `version` is obsolete 
[+] Running 2/1
 ✔ Container adguardhome   Removed                                                                                                                                               0.1s 
 ✔ Network docker_default  Removed                                                                                                                                               0.0s 
git checkout HEAD -- ./docker/conf/AdGuardHome.yaml
➜  terraform-provider-adguard git:(main) ✗ 
```

Can you please revert this change so we can see if the test here in GH are successful?